lib/menu.js

```diff
@@ -26,6 +28,32 @@ module.exports = function (app) {
               }
             })
           }
+        },
+        {
+          label: 'Export as GeoJSON..',
+          click: function (item, focusedWindow) {
+            dialog.showSaveDialog({
+              title: 'Guardar como..',
+              defaultPath: 'export.geojson',
+              filters: [{name: 'GeoJSON', extensions: ['geojson']}]
+            }, function (filename) {
+              if (!filename) return
+              var bbox = [-Infinity, -Infinity, Infinity, Infinity]
+              var out = fs.createWriteStream(filename)
+              var ex = exportGeoJson(app.osm, {bbox: bbox})
+              pump(ex, out, function (err) {
```

Comment:
I know I said I did not need to review this, but I have just realized that here we have a period when something is happening but the state is unclear. Export in my experience can take several seconds, whereas here it would appear to the user that the information is saved and the "exportación completa!" dialog would come as a bit of a surprise. We should show a dialog with some kind of progress, if it's not too hard. If this gets complicated, let's just stick with this for now - it's better having something working sooner than getting the UX perfect.